app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,48 @@
+class PostsController < ApplicationController
+  def index
+    # @current_user = current_user
+    @user = User.find(params[:user_id])
+    @post = Post.where(author_id: params[:user_id])
+    # @post = @user.posts
+
+    # @post = Post.includes(:comments: [:comments, { comments: [:author] }]).find(params[:user_id])
+
+    # @user = User.includes(:posts, posts: [:comments,
+    #  { comments: [:author] }]).find(params[:user_id])
+    # @posts = Post.where(author_id: params[:user_id]).includes(:user)
+    render '/posts/index'
+  end
+
```

Comment:
- [ ] One of the requirements is to solve the **_N+1 problems_** in this application. A good starting point is the `index` action of the `Posts` controller. Kindly solve the **_N+1 problem_** in this controller. You can read this [article](https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations:~:text=13%20Eager%20Loading%20Associations) to get a better understanding of how to fix this problem. 👍 💻